nnstreamer_example/custom_example_opencv/nnstreamer_customfilter_opencv_scaler.cc

```diff
@@ -0,0 +1,170 @@
+/**
+ * NNStreamer OpenCV Custom Filter Example: Scaler
+ * 
+ * Copyright (C) 2018 Sangjung Woo <sangjung.woo@samsung.com>
+ *
+ * LICENSE: LGPL-2.1
+ *
+ * @file  nnstreamer_customfilter_opencv_scaler.cc
+ * @date  22 Oct 2018
+ * @brief  OpenCV Custom NNStreamer Filter Example: Scaler
+ * @author  Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug  No known bugs
+ * @see  nnstreamer_customfilter_example_scaler_allocator.c
+ * 
+ * This example scales an input tensor of [N][input_h][input_w][M] 
+ * to an ouput tensor of [N][output_h][output_w][M].
+ *
+ * The custom property is to be given as, "custom=[new-x]x[new-y]", where new-x and new-y are unsigned integers.
+ * E.g., custom=640x480
+ *
+ */
+
+#include "opencv2/opencv.hpp"
```

Comment:
Is this element is compitable with OpenCV version 2 only? 
(Excudling OpenCV version 3?)

Reply:
No. IIRC, both OpenCV2 & OpenCV3 headers are in the _same_ path, which is `opencv2`. So this code supports both of them. Please refer below. 
* Link: https://docs.opencv.org/3.1.0/db/dfa/tutorial_transition_guide.html

> In 2.4 all headers are located in corresponding module subfolder (opencv2/<module>/<module>.hpp), in 3.0 there are top-level module headers containing the most of the module functionality: opencv2/<module>.hpp and all C-style API definitions have been moved to separate headers (for example opencv2/core/core_c.h).